internal/client/service.go

```diff
@@ -52,6 +54,19 @@ type CreateServiceRequest struct {
 	RegionCode   string
 	ReplicaCount string
 	VpcID        int64
+	ForkConfig   *ForkConfig
+}
+
+type ForkConfig struct {
+	ProjectID string `json:"projectID"`
```

Comment:
the input for ForkConfig when creating a service uses projectID, while the types for ForkSpec use projectId so I think I need different structs here.

Reply:
😢